lib/taskana-core-test/src/test/java/acceptance/jobs/AbstractTaskanaJobAccTest.java

```diff
@@ -131,6 +133,37 @@ void should_FindNoJobsToRunUntilFirstRunIsReached_When_CleanupScheduleIsInitiali
     }
   }
 
+  @WithAccessId(user = "admin")
+  @Test
+  void should_DeleteOldHistoryCleanupJobs_When_InitializingSchedule() throws Exception {
+
+    for (int i = 0; i < 10; i++) {
+      ScheduledJob job = new ScheduledJob();
+      job.setType(HistoryCleanupJob.class.getName());
+      taskanaEngine.getJobService().createJob(job);
+      job.setType(TaskRefreshJob.class.getName());
+      taskanaEngine.getJobService().createJob(job);
+      job.setType(ClassificationChangedJob.class.getName());
+      taskanaEngine.getJobService().createJob(job);
+    }
+
+    List<ScheduledJob> jobsToRun = jobMapper.findJobsToRun(Instant.now());
+
+    assertThat(jobsToRun).hasSize(30);
```

Comment:
This assertion should not be relevant for the test case. What do you think about removing it?

Reply:
You are right, I will remove it